build.gradle

```diff
@@ -42,6 +43,7 @@ subprojects {
     apply plugin: 'kotlin-kapt'
     apply plugin: 'kotlin'
     apply plugin: 'com.bmuschko.docker-remote-api'
+    apply plugin: 'com.diffplug.gradle.spotless'
```

Comment:
❓ Do you have to define this plugin twice? You're doing it in the plugins block (line 19) and here

Reply:
If I take it out I get the error
`Multiple steps with name 'ktlint' for spotless format 'kotlin'`